modules/statistics/CallStats.js

```diff
@@ -359,12 +359,24 @@ export default class CallStats {
             CallStats.callStatsID = options.callStatsID;
             CallStats.callStatsSecret = options.callStatsSecret;
 
+            let configParams;
+
+            if (options.applicationName) {
+                configParams = {
+                    applicationVersion:
+                        `${options.applicationName} (${
+                            RTCBrowserType.getBrowserName()})`
+                };
```

Comment:
What happens if this goes over the max of 30 characters?

Reply:
I got a response from callstats, they will truncate it and no error will be returned.